docs/modules/panel_configuration/pages/forecasting.adoc

```diff
@@ -1,104 +1,107 @@
-.. Consider the seasonality of the data when specifying a time range (for example, you may not want weekend network usage to be included in a forecast of usage during regular business hours).
-. If you have noisy or spiky data, forecasting does not work as well.
-You should also add the Chomp and Outlier filters, and move them so that they appear before the HoltWinters filter in the list of queries.
-You must specify an input for the outlier filter.
-. Save the dashboard.
+** Your time range must include at least more than twice the period size (in this case, 7200 seconds or more) to retrieve the historical data required for the forecast.
+** Consider the seasonality of your data when specifying a time range (for example, you may not want to include weekend network usage in a forecast of traffic during regular business hours).
+** Note that if your data is noisy or spiky, forecasting does not work as well.
+In these cases, you should add the `Chomp` and `Outlier` filters and rearrange them so they appear before the `HoltWinters` filter in the list of queries.
+You must specify an input for the `Outlier` filter.
+. Click *Save*.
 
 === Display forecast data
-Since the default time range for a panel returns data from the past to the present, you need to create a custom time range to display forecast data beyond the current time.
 
-. In the upper-right of the dashboard, select *Custom time range* from the time range drop-down.
-. Specify the start of your data range in the *From* field (for example, `now-6h`).
-. For the end time, specify a time in the future, as in the following:
-+
-image::pc-time-range.png[Range, 400]
+Because the default time range for a panel returns data from the past to the present, you must create a custom time range to display forecast data:
+
+. Select *Custom Time Range* from the *Time Range* list.
+. Specify the start of your time range in the *From* box (for example, `now-6h`).
+. Specify the end of your time range in the *To* box:
 +
-The panel displays the forecast you specified:
+image::pc-time-range.png["Example dashboard time range configuration.", 400]
+
+The panel displays the forecast data for the time period that you specify:
 +
-image::pc-forecast.png[Periods, 600]
+image::pc-forecast.png["Example panel displaying forecast data.", 700]
 
-. Experiment with changing the time range and forecast results.
-The further into the future you forecast, the less authoritative the forecast will be.
+You can experiment with the time range function to see how it updates your forecast results.
+Note that the further into the future you forecast, the less authoritative the data will be.
 
-[[predefined-filters]]
-=== Predefined filters
+== Filters
 
-The following table describes the pre-defined filters from OpenNMS that Helm provides for forecast metrics:
+The following predefined filters can be used to create forecasting metrics:
 
-[cols="15%,40%,45%"]
+[options="autowidth"]
```

Comment:
The filter column is too narrow, so the actual filter names are getting smushed, even at full screen.

Reply:
```suggestion
[cols="1,2,2"]
```